tests/conftest.py

```diff
@@ -54,7 +55,7 @@ def remove_subparser(name: str) -> None:
 
 
 @pytest.fixture
-def clean_argparser() -> callable:
+def clean_argparser() -> t.callable:
```

Comment:
I think it's t.Callable and not t.callable

Reply:
Yes, of course. Sorry abt that